main.go

```diff
@@ -29,9 +30,26 @@ type knownIssuesWorkflowInputSchema struct {
 }
 
 func openLinkInBrowser(link string) {
-	color.Green(fmt.Sprintf("Your new ticket has been successfully created! The link is %s \nWe will now try to open the new ticket for you in the browser. (Probably doesn't work on Windows.)", link))
-	cmd := exec.Command("open", link)
-	cmd.Run()
+	color.Green(fmt.Sprintf("Your new ticket has been successfully created! The link is %s \nWe will now try to open the new ticket for you in the browser.", link))
+	os := runtime.GOOS
+	switch os {
+	case "windows":
+		cmd := exec.Command("start", link)
+		cmd.Run()
+	case "darwin":
+		cmd := exec.Command("open", link)
+		cmd.Run()
+	case "linux":
+		out, err := exec.Command("which xdg-open").Output()
+		_ = out
```

Comment:
```suggestion
		_, err := exec.Command("which xdg-open").Output()
```

No need to capture `out` if you "throw it away" on the next line :)